src/client/src/containers/AzureLoginModal/strings.ts

```diff
@@ -0,0 +1,40 @@
+import { defineMessages } from "react-intl";
+
+export const strings = defineMessages({
+    freeAccountAnswer: {
+        defaultMessage: "All you need is a phone number, a credit or debit card, and a GitHub account or Microsoft account username (formerly Windows Live ID).",
+        id: "azureLoginModal.freeAccountAnswer"
+    },
+    freeAccountQuestion: {
+        defaultMessage: "What do I need to sign up for a free account?",
+        id: "azureLoginModal.freeAccountQuestion"
+    },
+    freeTrialUpgradeAnswer: {
+        defaultMessage: "TODO",
```

Comment:
@xtinah-w please help!

Reply:
the suggestions did not add automatically, I just manually added. PR updated :)